src/constants/analytics.ts

```diff
@@ -65,8 +67,8 @@ export const AnalyticsUserProperties = unionize(
     Network: ofType<DydxNetwork>(),
 
     // Wallet
-    WalletType: ofType<WalletType | null>(),
-    WalletConnectionType: ofType<WalletConnectionType | null>(),
+    WalletType: ofType<WalletType | string | null>(),
+    WalletConnectorType: ofType<ConnectorType | null>(),
```

Comment:
Since we are deprecating a field and adding a new one prob check to see if research/product need to update Amplitude dashboards (not sure what we can do in the case of backfilling)

Reply:
oo yess good callout